### PR TITLE
enhance(search): show placeholder panels for chart thumbnails

### PR DESCRIPTION
**Pull request by marcelgerber**
Currently, we have loads of layout jumps in the search UI while the chart previews load.
This is not much of an issue on fast networks, but is very noticeable on slow networks.

In the future, when we want to use dynamic thumbnails sometimes, we need to also think about
- that these take longer to load, and
- that they can also fail to load in rare cases.

That's why I've worked on a jump-less placeholder UI, and also a small error tile that we can show instead.

<table>
<tr>
<th>Before
<th>After
<tr>
 <td>

CleanShot 2024-03-20 at 09 45 10

 <td>

CleanShot 2024-03-20 at 09 47 05

<tr>
<td>Many layout jumps during loading
<td>No layout jumps; error tile for when errors occur

</table>

## Testing

For testing this:
- It's best to use DevTools' network throttling, for example "Fast 3G".
- To test the error behavior, manually clear out the `src` field of an `img` element in DevTools.

### CODE REVIEW

**Comment by marcelgerber** [comment]
* **#3371** <a href="https://app.graphite.dev/github/pr/owid/owid-grapher/3371?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @marcelgerber and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by marcelgerber** [comment]
For testing this:
- It's best to use DevTools' network throttling, for example "Fast 3G".
- To test the error behavior, manually clear out the `src` field of an `img` element in DevTools.

File: site/search/Search.scss [hunk 1]
```diff
@@ -271,16 +271,53 @@
...
+        opacity: 0;
+    }
+    to {
+        opacity: 1;
+    }
+}
+
 .search-results__chart-hit-img-container {
     margin-bottom: 8px;
+    aspect-ratio: 850 / 600;
```
Comment (marcelgerber) [inline site/search/Search.scss:285]: I'll just note here that [browser support for `aspect-ratio`](https://caniuse.com/mdn-css_properties_aspect-ratio) is currently 94% - it has been implemented in all major browsers in 2021.

If the user's browser doesn't support it, things will still work - the only thing that's happening is that the error tile will not be full-height.
I think this is absolutely acceptable behaviour for a small minority of browser share.

**Review by ikesau — APPROVED** [review]
Excellent.

**Pull request merged**